### PR TITLE
Feature/header

### PR DESCRIPTION
**Pull request by jamruthsai**
Added the header Component

### CODE REVIEW

**Comment by kprakhar04** [comment]
One little suggestion that try to keep your nav-links data in an array of objects(Header.js) and then using map function generate a list and render it at once instead of copying & pasting similar things. In my opinion it would be scalable & maintainable.

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,264 @@
-.App {
-  text-align: center;
+* {
+  margin: 0px;
```
Comment (kprakhar04) [inline src/App.css:2]: You can remove the units as we're specifying value as 0

File: src/components/Header/HeaderLinks.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import Link from './Link';
+function HeaderLinks(props) {
+  const { headerLinksConfig } = props;
+
+  return (
+    <>
+      <nav className='navbar'>
+        <ul>
+          {headerLinksConfig.map((item, index) => {
```
Comment (kprakhar04) [inline src/components/Header/HeaderLinks.js]: Move this iteration fn to a helper one (either in a separate file or above the functional component in the same file)

File: src/components/Header/Breadcrumb.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+function Breadcrumb(props) {
+  const { breadcrumbConfig } = props;
+  return (
+    <section className='breadcrumb'>
+      {breadcrumbConfig.map((item, index) => {
```
Comment (kprakhar04) [inline src/components/Header/Breadcrumb.js]: Move this out in a separate fn and just render the list

File: src/components/Header/constants/config.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
+import search from '../../../images/search.svg';
+import offers from '../../../images/offer.svg';
+import help from '../../../images/help.svg';
+import signIn from '../../../images/signin.svg';
+import cart from '../../../images/cart.svg';
+
+export const headerLinksConfig = [
```
Comment (mahendarsingh1) [inline src/components/Header/constants/config.js:7]: you can have two different files for both headerlinks and breadcrumb and export it by default

Comment (jamruthsai) [reply src/components/Header/constants/config.js:7]: Yeah, i will change it

**Pull request merged**